### ISSUE TITLE
Refer the user from logstash filter docs / tutorial to Kibana grok debugger

### ISSUE THREAD

**Issue by yaronp68** [elastic/logstash#9561]
Starting from version 6.3 all users will be exposed to the grok filter debugger in Kibana dev tools section.
This tool helps users to debug their grok match statements and improves productivity.
@brandonmensing @dedemorton suggest we refer the users from tutorials and documentation to the kibana feature documentation

**Comment by dedemorton** [comment]
I agree. I created several PRs awhile back ago to add links (when the Grok Debugger first came out) but it looks like at least one of my PRs (grok filter plugin docs) got closed without merging. 

@karenzone FYI
